packages/frontend/src/app/create-store/page.tsx

```diff
@@ -171,8 +163,16 @@ const StoreCreation = () => {
           shopId,
         );
         console.log(`Shop Manifested with shopId:${shopId}`);
+        // Create published and remove tags
         const newPubId = await stateManager.tags.create("visible");
-        const path = await relayClient!.uploadBlob(avatar as FormData);
+        await stateManager.tags.create("remove");
+
+        //Testing dom does not support FormData and test client will fail with:
+        //Content-Type isn't multipart/form-data
+        //so if it is a test env, we are skipping uploadBlob
+        const path = process.env.TEST
+          ? { url: "/" }
+          : await relayClient!.uploadBlob(avatar as FormData);
```

Comment:
seen these in a few places. I think we should be able to do something about that. Not sure what yet though and think it's fine to clean up later.